Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -165,7 +165,16 @@ public static Task<T> AwaitableRunAsync<T>(this CoreDispatcher dispatcher, Func<
              {
                  try
                  {
-                     taskCompletionSource.SetResult(await function().ConfigureAwait(false));
+                     var awaitableResult = function();
+                     if (awaitableResult != null)
+                     {
+                         var result = await awaitableResult.ConfigureAwait(false);
+                         taskCompletionSource.SetResult(result);
```

Comment:
I don't think so. Result can be null because this is `object` and it is used as a throw away object for the purpose of executing `void` type of function (`Action` type)

But then, at any given time, `function` can't be null, unless the caller especially pass `null` in for it.